symfony/webpack-encore-bundle/1.0/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "devDependencies": {
-        "@symfony/stimulus-bridge": "^1.1.0",
-        "@symfony/webpack-encore": "^0.32.0",
+        "@symfony/stimulus-bridge": "^1.2.0",
+        "@symfony/webpack-encore": "^1.0.0",
```

Comment:
Isn't this considering a BC break if we now require a new major version? Or does it contain no breaks?

Reply:
Yes, the new version does contain BC breaks. But this is the awkward things with the Encore recipe: the bundle does not have or need a new version. But we will want new projects to begin using `@symfony/webpack-encore` 1.0.